server/fetch-data.ts

```diff
@@ -19,7 +21,9 @@ export type DataType = {
   domain: string;
   upcomingEvents?: LocaleUpcomingEvent;
   customDomain?: string;
-};
+  profileMenu?: MenuContent[];
```

Comment:
is this necessary to add profileMenu here? I could very well all be in uiContent couldn't it? I don't wan't it to be optional and I wan't it to be static for every client for now, so create a defaultProfileMenu with your current fields in a new types file called "static-ui-content.ts", type it correctly and import it we're you need in the components instead of fetching it with the rest of the data

Reply:
so you want a default profilemenu type to get dthe defaut profile menu item like deconnexion and the rest to be another menu in uicontent?

Reply:
no, I when it to be separated from ui content and put in a static-ui-content, shared between all client